src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -22,6 +24,7 @@ public class ModelManager implements Model {
     private final AddressBook addressBook;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
+    private final FocusMode focusMode;
```

Comment:
Is a `FocusMode` class really necessary? It is essentially storing a Tag (later Class) object. 
The model class already has many functions that interact intermediate objects of the system. So wouldn't it be better to store a `Tag focusedClass` instead of the current `FocusMode` here, and move the functionality of `FocusMode` inside `ModelManager`?